velox/dwio/common/CachedBufferedInput.cpp

```diff
@@ -37,6 +37,12 @@ using cache::SsdPin;
 using cache::TrackingId;
 using memory::MemoryAllocator;
 
+void CachedBufferedInput::close() {
+  for (auto& load : allCoalescedLoads_) {
```

Comment:
Not sure if we could clear allCoalescedLoads_ here and have a check in ~CachedBufferedInput that allCoalescedLoads_ is empty?